Create .gitbook.yaml (#313)
mattermost/mattermost-plugin-incident-response: docs/incidents/starting-an-incident.md

# Starting an Incident

To start an incident, use one of the following steps:

- Use the slash command */incident start* from any channel.

> - Select the shield icon in the channel header, and click **+ Start Incident**

Reviewer: I'm ok with this, but technically, this can only be done from a channel that is not an incident channel. Also, a minor typo:
```suggestion
- Select the shield icon in the channel header, and click **+ Start Incident**.
```

- Use the context menu of a post and select **Start incident**.

> An interactive dialog appears prompting the selection of a playbook and channel name. Click **Start Incident** to create an incident with the selected playbook. Only playbook members can use the playbook to start an incident.

Reviewer: The interactive dialog reads "Incident name", not channel name. Although you're right that it's technically the channel name. Also, there's the optional description too, but I'm ok not polluting this paragraph with that.
![image](https://user-images.githubusercontent.com/3924815/95464509-51ebcc80-097a-11eb-9995-e2ffb98d5974.png)

> An interactive dialog appears prompting the selection of a playbook and channel name. Click **Start Incident** to create an incident with the selected playbook. Only playbook members can use the playbook to start an incident.

Author: ```suggestion
An interactive dialog appears prompting the selection of a playbook and incident name, which is required before you can select **Start Incident**. Only playbooks that you're a member of are listed in the **Playbook** drop-down menu.
```

The newly-created incident channel is the central place for discussion related to the incident. The incident bot announces the creator of the incident with a post in the channel. If an incident is started from the context menu of a post, the text of that post is included in the announcement message.

> The creator of a playbook is automatically added as the first member and becomes the commander. The commander is responsible for adding other members to the incident as needed, and may assign a new commander once a member is added to the channel. To change commanders, click the current commander's name in the RHS and select the new commander. Only members of the channel may be selected as commanders. To change commander to a user who is not in the channel, first add the user to the channel.

Reviewer: ```suggestion
The creator of an incident is automatically added as the first member and becomes the commander. The commander is responsible for adding other members to the incident as needed, and may assign a new commander once a member is added to the channel. To change commanders, click the current commander's name in the RHS and select the new commander. Only members of the channel may be selected as commanders. To change commander to a user who is not in the channel, first add the user to the channel.
```
